tests/helpers.py

```diff
@@ -37,11 +37,9 @@ def causal_lm_train_kwargs(train_kwargs):
         lora_config,
         prompt_tuning_config,
     ) = parser.parse_dict(train_kwargs, allow_extra_keys=True)
-    return (
-        model_args,
-        data_args,
-        training_args,
-        lora_config
-        if train_kwargs.get("peft_method") == "lora"
-        else prompt_tuning_config,
-    )
+    tuning_config = None
```

Comment:
hmm instead of this helper method, could we import build.utils and run `process_launch_training_args()` instead?

Reply:
we could, but is that not going away too after the refactor when we ll have a common function? ideally we shouldnt use anything and just create config classes themselves as purpose here is to test functionality of other things given arguments are correct . let me see how hard it would be to just get of this entire need to pass kwargs

Reply:
I was thinking of mentioning that this would be going away anyways soon, but I thought changing to use the import would be easy. If not then no worries